render_bundle/tests/unit/src/test_render_bundle.py

```diff
@@ -15,24 +15,26 @@ def file_content(file: str) -> str:
 
 
 class TestRenderBundle(unittest.TestCase):
-    def test_given_sdcore_up_when_render_bundle_then_bundle_is_rendered(self):
-        rendered_bundle_path = f"{TEST_PATH}/rendered_bundle_sdcore_up.yaml"
-        expected_bundle_path = f"{TEST_PATH}/expected_bundle_sdcore_up.yaml"
+    def test_given_sdcore_user_plane_when_render_bundle_then_bundle_is_rendered(self):
+        rendered_bundle_path = f"{TEST_PATH}/rendered_bundle_sdcore_user_plane.yaml"
+        expected_bundle_path = f"{TEST_PATH}/expected_bundle_sdcore_user_plane.yaml"
         render_bundle(
             local=False,
-            bundle_variant="SDCORE_UP",
+            bundle_variant="SDCORE_USER_PLANE",
+            channel="beta",
             output_file=rendered_bundle_path,
         )
 
         self.assertEqual(file_content(rendered_bundle_path), file_content(expected_bundle_path))
 
-    def test_given_sdcore_up_local_when_render_bundle_then_bundle_is_rendered(self):
-        rendered_bundle_path = f"{TEST_PATH}/rendered_bundle_sdcore_up_local.yaml"
-        expected_bundle_path = f"{TEST_PATH}/expected_bundle_sdcore_up_local.yaml"
-        render_bundle(
-            local=True,
-            bundle_variant="SDCORE_UP",
-            output_file=rendered_bundle_path,
-        )
 
-        self.assertEqual(file_content(rendered_bundle_path), file_content(expected_bundle_path))
+#     def test_given_sdcore_user_plane_local_when_render_bundle_then_bundle_is_rendered(self):
+#         rendered_bundle_path = f"{TEST_PATH}/rendered_bundle_sdcore_user_plane_local.yaml"
+#         expected_bundle_path = f"{TEST_PATH}/expected_bundle_sdcore_user_plane_local.yaml"
+#         render_bundle(
+#             local=True,
+#             bundle_variant="SDCORE_USER_PLANE",
+#             output_file=rendered_bundle_path,
+#         )
+#
+#         self.assertEqual(file_content(rendered_bundle_path), file_content(expected_bundle_path))
```

Comment:
Do we want to keep this commented out?

Reply:
Oups no, this was an artifact from testing. Now fixed.